### PR TITLE
task 2.5 created class

### PR DESCRIPTION
**Pull request by YanaBocharova**
5) Given a string s, reverse the order of characters in each word within a sentence while still preserving whitespace and initial word order.

 

Example 1:

Input: s = "Let's take LeetCode contest"
Output: "s'teL ekat edoCteeL tsetnoc"
Example 2:

Input: s = "God Ding"
Output: "doG gniD"

### CODE REVIEW

File: AlgoritmicalTasksAndUT/AlgoritmicalTasksAndUT/ReverseElementString.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+using System.Linq;
+using System.Text;
+
+namespace AlgoritmicalTasksAndUT
+{
+    public class ReverseElementString
+    {
+        public string ReverceString(string str)
+        {
+            List<string> s = str.Split(' ').ToList();
```
Comment (chocolatapie) [inline AlgoritmicalTasksAndUT/AlgoritmicalTasksAndUT/ReverseElementString.cs:12]: give `s` a meaningful name

File: AlgoritmicalTasksAndUT/AlgoritmicalTasksTests/ReverseElementStringTest.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    {
+        [TestMethod]
+        public void StringConsistentAnycharactersTest_InputStringt_ReturnReverseString()
+        {
+            // arrange 
+            var TestObj = new  ReverseElementString();
+
+            string str = ("Good word");
+
+            string reverseStr = ("dooG drow ");
```
Comment (chocolatapie) [inline AlgoritmicalTasksAndUT/AlgoritmicalTasksTests/ReverseElementStringTest.cs:21]: there is an extra space at the end of word

File: AlgoritmicalTasksAndUT/AlgoritmicalTasksTests/ReverseElementStringTest.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+        [TestMethod]
+        public void StringConsistentAnycharactersTest_InputEmptyStringt_ReturnEmptyString()
+        {
+            // arrange 
+            var TestObj = new ReverseElementString();
+
+            string str = string.Empty;
+
+            string reverseStr = " ";
```
Comment (chocolatapie) [inline AlgoritmicalTasksAndUT/AlgoritmicalTasksTests/ReverseElementStringTest.cs:38]: should return empty string, not a space

File: AlgoritmicalTasksAndUT/AlgoritmicalTasksTests/ReverseElementStringTest.cs (hunk 2 above)
Comment (YanaBocharova) [reply AlgoritmicalTasksAndUT/AlgoritmicalTasksTests/ReverseElementStringTest.cs]: fixed

File: AlgoritmicalTasksAndUT/AlgoritmicalTasksAndUT/ReverseElementString.cs (hunk 1 above)
Comment (YanaBocharova) [reply AlgoritmicalTasksAndUT/AlgoritmicalTasksAndUT/ReverseElementString.cs]: fixed

**Review by chocolatapie — APPROVED** [review]

**Pull request merged**